src/components/slide-carousel/README.md

```diff
@@ -2,7 +2,6 @@
 
 #### SlideCarousel
 ```
-const SlideCarousel = require('./SlideCarousel').default;
 const Slide = require('./Slide').default;
```

Comment:
QQ: what makes it ok to delete some of these but not others?

Reply:
It seems functional components need to be imported somewhere while class components don't when they are only used once.

Reply:
huh that's weird behavior. although not really worth looking into since we're aiming to remove styleguidist soon